src/libraries/System.Private.Xml/src/System/Xml/Serialization/XmlSerializer.cs

```diff
@@ -199,6 +199,12 @@ public XmlSerializer(Type type, string? defaultNamespace)
                 _primitiveType = type;
                 return;
             }
+
+            if (Mode == SerializationMode.ReflectionOnly)
+            {
+                return;
```

Comment:
Do we need to also execute `GetTopLevelMapping` at the very bottom of the method in ReflectionOnly mode?

Reply:
No, I think this was all ifdeffed out:

https://github.com/dotnet/corefx/blob/e31a2b7f3bdf985f56798df530daa4c06fd80376/src/System.Private.Xml/src/System/Xml/Serialization/XmlSerializer.cs#L266-L308